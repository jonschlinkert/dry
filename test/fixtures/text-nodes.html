{% extends "basic-text" %}

{% block "header" %}
header: from text-nodes.html
{% endblock %}

This is outside of a block.
