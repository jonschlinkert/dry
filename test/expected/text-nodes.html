
header: from text-nodes.html
text node from "blocks/basic.html"